config/environments/test.rb

```diff
@@ -64,4 +64,6 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
+
+  config.factory_bot.definition_file_paths = ["spec/support/factories"]
```

Comment:
looks good although it might be also considered ok to use `Rails.root.join(%w[spec support factories])` using the `join` would be OS (Operating System) agnostic (ie work on windows with spec\support\factories) as well - also `Rails.root` would make it more obvious where this directory would live

all that said - I would probably stick with what you have got